Gemfile.lock

```diff
@@ -616,65 +618,63 @@ GEM
     sawyer (0.9.2)
       addressable (>= 2.3.5)
       faraday (>= 0.17.3, < 3)
-    sentry-rails (5.4.2)
+    sentry-rails (5.16.1)
       railties (>= 5.0)
-      sentry-ruby (~> 5.4.2)
-    sentry-ruby (5.4.2)
+      sentry-ruby (~> 5.16.1)
+    sentry-ruby (5.16.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
     sentry-user_informer (0.2.0)
       rack
       railties (>= 6.1.0, < 7.1.0)
       sentry-rails (~> 5.4)
-    sexp_processor (4.12.1)
-    single_cov (1.3.2)
-    slop (3.6.0)
+    sexp_processor (4.17.1)
+    single_cov (1.11.0)
     snaky_hash (2.0.1)
       hashie
       version_gem (~> 1.1, >= 1.1.1)
     socksify (1.7.1)
-    soft_deletion (1.6.0)
-      activerecord (>= 4.2.0, < 6.2.0)
+    soft_deletion (1.9.0)
+      activerecord (>= 5.0.0, < 7.2)
     sprockets (3.7.2)
       concurrent-ruby (~> 1.0)
       rack (> 1, < 3)
     sprockets-rails (3.4.2)
       actionpack (>= 5.2)
       activesupport (>= 5.2)
       sprockets (>= 3.0.0)
-    sqlite3 (1.4.2)
-    stackprof (0.2.12)
+    sqlite3 (1.7.2)
+      mini_portile2 (~> 2.8.0)
+    stackprof (0.2.26)
     terminal-table (1.8.0)
       unicode-display_width (~> 1.1, >= 1.1.1)
     thor (1.3.0)
-    tilt (2.0.10)
+    tilt (2.3.0)
     tzinfo (2.0.6)
       concurrent-ruby (~> 1.0)
     uglifier (3.2.0)
       execjs (>= 0.3.0, < 3)
-    unf (0.1.4)
-      unf_ext
-    unf_ext (0.0.8.2)
     unicode-display_width (1.8.0)
     validates_lengths_from_database (0.8.0)
       activerecord (>= 4)
     version_gem (1.1.3)
-    warden (1.2.7)
-      rack (>= 1.0)
-    webmock (3.0.1)
-      addressable (>= 2.3.6)
+    warden (1.2.9)
+      rack (>= 2.0.9)
+    webmock (3.20.0)
+      addressable (>= 2.8.0)
       crack (>= 0.3.2)
-      hashdiff
+      hashdiff (>= 0.4.0, < 2.0.0)
     websocket-driver (0.7.6)
       websocket-extensions (>= 0.1.0)
     websocket-extensions (0.1.5)
-    zeitwerk (2.6.12)
+    zeitwerk (2.6.13)
```

Comment:
### [zeitwerk](https://github.com/fxn/zeitwerk)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.6.12 → 2.6.13

[_[change-log](https://github.com/fxn/zeitwerk/blob/master/CHANGELOG.md), [source-code](https://github.com/fxn/zeitwerk), [gem-diff](https://my.diffend.io/gems/zeitwerk/2.6.12/2.6.13)_]





<details>
<summary>Commits</summary>

A change of **13** commits. See the full changes on [the compare page](https://github.com/fxn/zeitwerk/compare/v2.6.12...v2.6.13).

These are the first 10 commits:
- (74a2375) [Provide a null inflector](https://github.com/fxn/zeitwerk/commit/74a23756853098f265d8b03aa6743ad9ce00402a)
- (a829ff8) [Revise docs re the null inflector and Rails](https://github.com/fxn/zeitwerk/commit/a829ff81a2a7845e26e47d7d4a1bfe3ec16c8e91)
- (f381566) [Add Ruby 3.3 to CI matrix](https://github.com/fxn/zeitwerk/commit/f381566350681d7d35adbd7fd2c6d966993bd007)
- (b82324b) [Merge pull request #278 from m-nakamura145/update-ci-matrix](https://github.com/fxn/zeitwerk/commit/b82324bd03375e788f047d5b66fd429f78b4362b)
- (ff13527) [Fix small typo & clarify some grammar in the docs](https://github.com/fxn/zeitwerk/commit/ff13527ca1fb7b1edcb92852dcec55bd13d257f4)
- (a7201b3) [Merge pull request #279 from stevenharman/typo_and_doc_tweek…](https://github.com/fxn/zeitwerk/commit/a7201b3d756f341a1d823cf4e7d34ff623e1b17d)
- (94e83ee) [Delete comment in Kernel#require decoration](https://github.com/fxn/zeitwerk/commit/94e83ee90e14eee69a151a7b63e44bb58ade3ba3)
- (7daca61) [Revise the conceptual translation of for_gem](https://github.com/fxn/zeitwerk/commit/7daca616ff12f4e8eaf7a36649c4ca78082aa154)
- (f3cca26) [Bump actions/checkout](https://github.com/fxn/zeitwerk/commit/f3cca26dca9f258a149d0de1daccc9f2eeab5cce)
- (798ccdb) [Merge pull request #286 from m-nakamura145/update-actions-ch…](https://github.com/fxn/zeitwerk/commit/798ccdbecbb67197ee9befef3f19c0bf93b87561)


</details>